src/actions/index.js

```diff
@@ -7,3 +18,41 @@ export const fetchContacts = () => ({
 export const fetchFavourites = () => ({
   type: FETCH_FAVOURITES
 });
+
+export const increaseLikes = id => ({
+  type: INCREASE_LIKES,
+  payload: id
+});
+
+export const decreaseLikes = id => ({
+  type: DECREASE_LIKES,
+  payload: id
+});
+
+export const changeIsChecked = id => ({
+  type: CHANGE_IS_CHECKED,
+  payload: id
+});
+
+export const deleteContact = id => ({
+  type: DELETE_CONTACT,
+  payload: id
+});
+
+export const addContact = formValues => ({
+  type: ADD_CONTACT,
+  payload: {
+    ...formValues,
```

Comment:
`actions` should not have any logic. Only send `formValues` as payload and create the new object with the required fields in the reducer.